_posts/2025-06-30-probabl-skolar.md

```diff
@@ -0,0 +1,93 @@
+---
+#### Blog Post Template ####
+
+#### Post Information ####
+title: "Skolar: an open-source initiative to democratize open data science"
+date: June 30, 2025
+
+#### Post Category and Tags ####
+# Format in titlecase without dashes (Ex. "Open Source" instead of "open-source")
+categories:
+  - Updates
+tags:
+  - Sponsor
+  - Open Source
+  - Machine Learning
+
+#### Featured Image ####
+featured-image: BSD_watermark.svg
+
+#### Author Info ####
+# Can accomodate multiple authors
+# Add SQUARE Author Image to /assets/images/author_images/ folder
+postauthors:
+  - name: Skolar
+    website: https://skolar.probabl.ai/
+    image: "skolar-logo.png"
+  - name: Pénélope Gittos
+    website: https://www.linkedin.com/in/gittospenelope-data-analyst-growth-bilingual/
+    image: "penelope_gittos.jpeg"
+---
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+
+<span style="color:red">*This blog post has been submitted by Probabl, a sponsor of scikit-learn.* </span>
+The scikit-learn project always puts efforts on education to build and nurture a
+strong vibrant open-source community. The goal is straightforward: give
+everyone, everywhere, the tools they need to easily grasp, engage with, and
+meaningfully contribute to data science using open-source software. This mission
+is shared and actively supported by [Probabl](https://probabl.ai/), a company
+that helps maintain scikit-learn by employing many of its core contributors and
+investing in its long-term sustainability. With their support and a deep
+commitment from the community, we continue building bridges between research,
```

Comment:
"... With their support ... , we continue"

Maybe we can make it more explicit who is who here. At least I'm a bit lost if for example the "their" is the group of engineers employed by probabl, or probal or someone else? If it is probabl I think "With probabl's support and ..." would be clearer